jerry-core/ecma/builtin-objects/ecma-builtin-bigint.c

```diff
@@ -36,6 +52,289 @@
  * @{
  */
 
+/**
+ * The BigInt object's 'asIntN' and 'asUintN' routines
+ *
+ * See also:
+ *          ECMA-262 v5, 11.0
```

Comment:
v5? Is this really that old? Did you check the standard?

Reply:
BigInt introduced in ES2020 (11th edition). Please reference to the latest ES2024 (15th edition) with proper section number.